lib/resources/r4/related_person_patch.yaml

```diff
@@ -92,6 +92,34 @@ operations:
         <li>If `line` is provided, only the first four values are used. Additional values will be ignored.</li>
       </ul>
 
+  - name: test-address-id
+    path: /address/{index}/id
+    operation: test
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+    description: Test the `Address.id` value of the address at the given <code>{index}</code> in the list of related person addresses.
+    example: |
+      {
+        "op": "test",
+        "path": "/address/0/id",
+        "value": "CI-24079465-0"
+      }
+
+  - name: remove-address
+    path: /address/{index}
+    operation: remove
+    type: N/A
+    url: https://hl7.org/fhir/R4/patient-definitions.html#RelatedPerson.address
```

Comment:
I will add a comment for myself on a deeper look. This url is not correct.
https://github.com/cerner/fhir.cerner.com/commit/4e1538278c3b7a725fd27b0b4e3d2e5e9315d680#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR112

Reply:
Changed to the correct one: 
4e15382#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR112